.github/workflows/tests.yml

```
@@ -66,7 +66,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        version: [3.8, 3.9.18, 3.10.13, 3.11, 3.12]
+        version: [3.9.18, 3.10.13, 3.11, 3.12]
```

```suggestion
        version: [3.9, 3.10, 3.11, 3.12]
```
do you need to specify the patch version for some reason ?

---

yeah, it blows up installing 3.10 etc and claims only `3.10.13` available, pinning shuts it up